packages/manager/modules/netapp/src/index.js

```diff
@@ -0,0 +1,32 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+import template from './template.html';
+
+const moduleName = 'ovhManagerNetAppLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('netapp', {
+      url: '/netapp',
+      template,
+      redirectTo: 'netapp.index',
+      resolve: {
+        breadcrumb: () => 'netapp',
```

Comment:
are you planning to translate this later?

Reply:
For the moment, a translation is added. For the PR context, no translation demand will be done. We will wait for other netapp features, to check which content should be requested for translation.